### DISCUSSION TITLE
Banco de imagens

### DISCUSSION THREAD

**Discussion by christiansousadev** [EasyRoomCompany/Guardian#71] [Ideas]
Quem poder ajudar me enviando algumas imagens para eu ilustrar um pouco o nosso produto. Um layout de um sistema em funcionamento ou algo do tipo...

**Comment by mikaelmonteirodev** [comment]
https://github.com/HackathonTeam05/Guardian/assets/112489130/b9a0858d-fc07-401f-bd1f-efcd773bde02
